openc3-cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -396,6 +396,7 @@ def self.spawn(scope, name, suite_runner = nil, disconnect = false, environment
     process.environment['RUBYOPT'] = nil # Removes loading bundler setup
     process.environment['OPENC3_SCOPE'] = scope
 
+    process.detach = true
```

Comment:
This prevents zombie processes?

Reply:
Yes. Detaching means that the spawning process is not responsible for waiting for the child to die and check the exit status.